examples/credential_metadata_sd_jwt_vc.json

```
@@ -54,19 +56,17 @@
             }
           ]
         },
-        "email": {},
-        "phone_number": {},
-        "address": {
-          "street_address": {},
-          "locality": {},
-          "region": {},
-          "country": {}
-        },
-        "birthdate": {},
-        "is_over_18": {},
-        "is_over_21": {},
-        "is_over_65": {}
-      }
+        {"path": ["email"]},
+        {"path": ["phone_number"]},
```

```suggestion
        {"path": ["phone_number"]},
        {
          "path": ["address"],
          "display": [
            {
              "name": "Place of residence",
              "locale": "en-US"
            },
            {
              "name": "Wohnsitz",
              "locale": "de-DE"
            }
          ]
        }
        {"path": ["address", "street_address"]},
```

Add display information to non-leaf claim, as discussed in https://github.com/openid/OpenID4VCI/pull/276#discussion_r1497169863